src/io/flutter/settings/FlutterSettings.java

```diff
@@ -219,6 +223,16 @@ public void setShowWebDesktopDevices(boolean value) {
     fireEvent();
   }
 
+  public boolean isShowStructuredErrors() {
+    return getPropertiesComponent().getBoolean(showStructuredErrors, false);
```

Comment:
Should the default here be `true` to match the previous behavior?

Reply:
We didn't have this setting previously; the `false` will mean that the setting is not enabled unless the user explicitly toggles it. We'll keep it like that for a month or two, then change the default to `true` (and eventually, possibly remove the setting completely).